packages/universe-components/src/profile/ProfileForm.tsx

```diff
@@ -94,21 +59,68 @@ export default async function Profile({
   const stories = userData.stories;
   const nopic = '/media/nopic.png';
 
+  async function saveImage(file: File) {
+    console.log('IMAGE', file)
+    //if(file){ return {error:'no image provided'} }
+    const name = randomString(10)
```

Comment:
I think we should probably use uuidv7 to avoid collisions

Reply:
There are 63,403,380,965,376 combinations in 10 chars, I think we're safe for the next million years, and it's shorter